[WARNING] [2020-10-29 08:06:03,486] [2233:140187704457024] - evaluation_param.py[line:127]: use default metric ['auc', 'ks', 'lift', 'gain', 'accuracy', 'precision', 'recall', 'roc', 'confusion_mat', 'psi', 'f1_score', 'quantile_pr'] for eval type binary
[INFO] [2020-10-29 08:06:03,487] [2233:140187704457024] - evaluation_param.py[line:131]: Finish evaluation parameter check!
[DEBUG] [2020-10-29 08:06:03,487] [2233:140187704457024] - component_properties.py[line:67]: {'EvaluateParam': {'eval_type': 'binary', 'pos_label': 1, 'need_run': True, 'metrics': None, 'default_metrics': {'binary': ['auc', 'ks', 'lift', 'gain', 'accuracy', 'precision', 'recall', 'roc', 'confusion_mat', 'psi', 'f1_score', 'quantile_pr'], 'multi': ['accuracy', 'precision', 'recall'], 'regression': ['explained_variance', 'mean_absolute_error', 'mean_squared_error', 'median_absolute_error', 'r2_score', 'root_mean_squared_error']}, 'allowed_metrics': {'binary': ['auc', 'ks', 'lift', 'gain', 'accuracy', 'precision', 'recall', 'roc', 'confusion_mat', 'psi', 'f1_score', 'quantile_pr'], 'multi': ['accuracy', 'precision', 'recall'], 'regression': ['explained_variance', 'mean_absolute_error', 'mean_squared_error', 'median_absolute_error', 'r2_score', 'root_mean_squared_error']}}, 'initiator': {'role': 'guest', 'party_id': 9999}, 'job_parameters': {'work_mode': 0, 'processors_per_node': 1, 'align_task_input_data_partition': True, 'model_id': 'arbiter-10000#guest-9999#host-10000#model', 'model_version': '202010290805128735466'}, 'role': {'guest': [9999], 'host': [10000], 'arbiter': [10000]}, 'local': {'role': 'host', 'party_id': 10000}, 'CodePath': 'federatedml/evaluation/evaluation.py/Evaluation', 'module': 'Evaluation'}
[DEBUG] [2020-10-29 08:06:03,487] [2233:140187704457024] - component_properties.py[line:74]: need_run: True, need_cv: False
[DEBUG] [2020-10-29 08:06:03,487] [2233:140187704457024] - component_properties.py[line:118]: has_train_data: False, has_eval_data: False, has_normal_data: True
[DEBUG] [2020-10-29 08:06:03,487] [2233:140187704457024] - model_base.py[line:87]: func: <bound method Evaluation.fit of <federatedml.evaluation.evaluation.Evaluation object at 0x7f7ae42adf28>>
[DEBUG] [2020-10-29 08:06:03,487] [2233:140187704457024] - model_base.py[line:99]: saved_result is : [None], data_output: None
